api/src/main/java/org/pmiops/workbench/workspaces/WorkspacesController.java

```diff
@@ -141,6 +161,13 @@ public WorkspacesController(
       BillingProjectBufferService billingProjectBufferService,
       WorkspaceService workspaceService,
       CdrVersionDao cdrVersionDao,
+      CohortMapper cohortMapper,
```

Comment:
The introduction of 7 new services at once makes me think perhaps there's an opportunity to define 2-3 high-level intermediate services.

We're going to break all of @ericsong's workspace tools again by increasing the argument list, and testing is going to get increasingly hairy.

Reply:
I think that sort of new service should be a new PR. I am incredibly worried about the amount of scope creep happening on this ticket.